READMEs/fixed-rate-exchange-flow.md

```diff
@@ -0,0 +1,162 @@
+<!--
+Copyright 2021 Ocean Protocol Foundation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Quickstart: Fixed Rate Exchange Flow
+
+It focuses on Alice's experience as a publisher, and Bob's experience as a buyer & consumer.
+
+Here are the steps:
+
+1.  Setup
+2.  Alice creates a data token
+3.  Alice mints & approves data tokens
+4.  Bob buys at fixed rate data tokens
+
+Let's go through each step.
+
+## 1. Setup
+
+### Prerequisites
+
+-   Linux/MacOS
+-   Docker, [allowing non-root users](https://www.thegeekdiary.com/run-docker-as-a-non-root-user/)
+-   Python 3.8.5+
+
+### Run barge services
+
+In a new console:
+
+```console
+#grab repo
+git clone https://github.com/oceanprotocol/barge
+cd barge
+
+#clean up old containers (to be sure)
+docker system prune -a --volumes
+
+#run barge: start ganache, Provider, Aquarius; deploy contracts; update ~/.ocean
+./start_ocean.sh  --with-provider2
```

Comment:
do we need provider2?

Reply:
I've tested with it and it is included in marketplace-flow readme too